plugins/times-of-day/api/api.js

```diff
@@ -0,0 +1,187 @@
+var plugin = {},
+	common = require('../../../api/utils/common.js'),
+	plugins = require('../../pluginManager.js'),
+	async = require('async');
+
+(function (plugin) {
+	plugins.register("/i", function (ob) {
+		var params = ob.params;
+
+		if (!params.qstring.begin_session && !params.qstring.events) {
+			return false;
+		}
+
+
+		var appId = params.app_id;
+		var hour = params.qstring.hour;
+		var dow = params.qstring.dow;
+		var events = [];
+
+		if (!appId ||
+			!(parseInt(hour) >= 0 && parseInt(hour) <= 23) ||
+			!(parseInt(dow) >= 0 && parseInt(dow) <= 6)) {
+			return false;
+		}
+
+		var hasSession = false;
+		var hasEvents = false;
+
+		if (params.qstring.begin_session) {
+			hasSession = true;
+		}
+
+		if (params.qstring.events && params.qstring.events.length) {
+			hasEvents = true;
+			events = params.qstring.events;
+		}
+
+		var queryArray = [];
+		var query = {};
+		var criteria = {};
+		var update = {};
+		var options = {};
+
+		if (hasSession) {
+			criteria = {
+				"_id": "tod_" + appId + ":Sessions"
```

Comment:
Can we use [CLY]_session here for consistency with other places where session is used?